user_guide/intro_nav_user_menu.rst

```diff
@@ -0,0 +1,58 @@
+.. _user-guide-getting-started-user-menu:
+
+
+User Menu 
+=========
+
+On the right side of the screen under your username you will find the User Menu drop-down. It provides a fast way to 
+access your user profile, calendar, mailbox and task list. It is also used to log out of the system.
```

Comment:
Edit: On the right side of the screen, under your username, you will find the User Menu drop-down. It provides a fast way to access your user profile, calendar, mailbox, and task list. It is also used to log out of the system.